relay-monitors/src/lib.rs

```diff
@@ -176,7 +189,19 @@ pub fn process_check_in(payload: &[u8]) -> Result<Vec<u8>, ProcessCheckInError>
         return Err(ProcessCheckInError::InvalidEnvironment);
     }
 
-    Ok(serde_json::to_vec(&check_in)?)
+    static NAMESPACE: OnceCell<Uuid> = OnceCell::new();
+    let namespace = NAMESPACE
+        .get_or_init(|| Uuid::new_v5(&Uuid::NAMESPACE_URL, b"https://sentry.io/crons/#did"));
```

Comment:
Wasn't sure what the best namespace URL would be to use here. cc @mitsuhiko any thoughts?